plugins/usbdmx/AsyncUsbReceiver.cpp

```diff
@@ -58,25 +61,20 @@ bool AsyncUsbReceiver::Start() {
     return false;
   }
   ola::thread::MutexLocker locker(&m_mutex);
-  return PerformTransfer();
+  bool success = true;
+  while (!m_idle.empty() && success) {
+    success = success && PerformTransfer();
+  }
+  return success;
```

Comment:
Wouldn't this work just the same? As soon as success becomes false, because a transfer fails, it won't enter into the next loop and would then return false:
```suggestion
  bool success = true;
  while (!m_idle.empty() && success) {
    success = PerformTransfer();
  }
  return success;
```